lib/src/lib/services/form-builder.service.ts

```diff
@@ -82,7 +82,7 @@ export class Lab900FormBuilderService {
           field.options.defaultValue !== null &&
           typeof field.options.defaultValue !== 'undefined'
         ) {
-          controlValue = typeof field.options.defaultValue === 'function' ? field.options.defaultValue() : field.options.defaultValue;
+          controlValue = typeof field.options.defaultValue === 'function' ? field.options.defaultValue(data) : field.options.defaultValue;
```

Comment:
@lab900-frederico-teixeira What does this change?

Reply:
injects the data into the defaultValue so we can make some logic based on the existing data

Reply:
Ok, seems fine to me